src/plugins/intel_gpu/tests/unit/test_cases/convolution_gpu_test.cpp

```diff
@@ -8780,10 +8780,11 @@ TEST_P(convolution_random_smoke_test, u8s8f32_scale) {
     ASSERT_NO_FATAL_FAILURE(test.run_random(GetParam()));
 }
 
-TEST_P(convolution_random_smoke_test, s8s8f32_fsv4_input) {
-    convolution_random_test_fsv4_input_s8s8f32 test;
-    ASSERT_NO_FATAL_FAILURE(test.run_random(GetParam()));
-}
+// Block until issue fix of in-correct IMAD after SHUFFLE
+// TEST_P(convolution_random_smoke_test, s8s8f32_fsv4_input) {
+//     convolution_random_test_fsv4_input_s8s8f32 test;
+//     ASSERT_NO_FATAL_FAILURE(test.run_random(GetParam()));
+// }
```

Comment:
Please block the cases only for dGPU. We need to test iGPU still.

Reply:
It is fixed.